### DISCUSSION TITLE
Integration with Google calendar

### DISCUSSION THREAD

**Discussion by egahmad** [wdelfuego/nova-calendar#40] [Ideas & Feature Requests]
It would be nice to integrate Google calendar

**Comment by jeremiahsherrill** [comment]
that would be amazing

**Comment by wdelfuego** [comment]
I don’t think automatically displaying google calendar data would be too difficult. I won’t get around to it before Q1 ‘23 at the earliest though. 

A different thing would be support for editing events, as that is something not currently in this package.

Without specifying a timeline, my plan is to implement at least a week view and event filtering, then add display of google calendar data and finally think about event editing directly from the calendar view.
